exercises/binary-search/src/example/java/BinarySearch.java

```diff
@@ -0,0 +1,40 @@
+
+import java.util.List;
+
+public class BinarySearch<T extends Comparable<T>> {
```

Comment:
Since there are other files in `exercises/binary-search/src/main/java/` and `exercises/binary-search/src/test/java/`, it should be safe to remove `exercises/binary-search/src/main/java/.keep` and `exercises/binary-search/src/test/java/.keep`. Thanks for checking!